.github/workflows/build-site.yml

```diff
@@ -29,7 +29,7 @@ jobs:
           key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
           restore-keys: |
             ${{ runner.os }}-node-
-      - run: npm ci
+      - run: npm install
```

Comment:
Why do we need to change?

Reply:
ci command failed for some reason. It said requires package-lock file in sync with package.json, but I did run npm install.

Reply:
It seems we should use ci though, could you try to run npm install and see if you get a different lock file?